src/TextField/TextField.tsx

```diff
@@ -8,6 +8,11 @@ import { IconInfoSolid } from "../icons/IconInfoSolid";
 import classnames from "classnames";
 
 interface Props {
+  /**
+   * Passed through to the underlying `input`
+   */
+  autoFocus?: boolean;
```

Comment:
nbd: Maybe put this down below with the other props intended for input

Reply:
I'm going to merge this as-is because I need it ASAP; I'll add more tests after we talk about it :)